apps/dashboard/src/components/conditions-editor/variable-select.tsx

```diff
@@ -102,7 +102,14 @@ export const VariableSelect = (props: VariableSelectProps) => {
   };
 
   return (
-    <Popover open={isOpen}>
+    <Popover
+      open={isOpen}
+      onOpenChange={(open) => {
+        if (!open) {
+          onClose();
+        }
```

Comment:
ensures that when on click outside of the popover, it will be closed.

Reply:
Isn't this behavior controlled via the modal prop? https://www.radix-ui.com/primitives/docs/components/popover#root

Reply:
it is, and it works for conditions, but it did not worked for the digets key section